content/physical_properties/magnetic_permeability/magnetic_permeability_frequency_dependent.rst

```diff
@@ -16,7 +16,6 @@ Here, our description of the magnetic viscosity is limited to providing useful
 expressions for implementation in applied problems. Comprehensive studies of
 magnetic viscosity, and its relevance to geophysical surveying, may be found
 in (:cite:`Neel1949,Buselli1982,Lee1984,Dabasetal1992,DunlopOzdemir1997,Das2006,Pasion2007`).
-page.
```

Comment:
below we link the word `earlier` to the Magnetism page. Would you mind replacing that with the title of the page (or something close to the title). What a link contains is much more transparent if it is descriptive